bson/src/main/org/bson/json/JsonString.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bson.json;
+
+/**
+ * A wrapper class that holds a JSON string. This class makes decoding straight
+ * to JSON easy.
+ *
+ * @since 4.2
+ */
+public class JsonString {
```

Comment:
1) I wasn't sure where to put this file. This folder contains json stuff, but it's not that related to the purpose of this file. I was debating between this folder and its parent folder.

2) I decided against providing equals and hashcode methods because I thought users shouldn't really keep JsonStrings, they should just use them to access the string. Do you agree with this decision?

3) I don't have any unit tests for this class. Is this okay for simple wrapper classes like this?

Reply:
Done